生成启动时间为 2018/10/17 7:24:00。
     1>项目“C:\Users\hp\Documents\GitHub\2018.10.7\第三次\Project7\Project7\Project7.vcxproj”在节点 2 上(Build 个目标)。
     1>ClCompile:
         E:\Cpp\VC\bin\CL.exe /c /ZI /nologo /W3 /WX- /sdl /Od /Oy- /D _MBCS /Gm /EHsc /RTC1 /MDd /GS /fp:precise /Zc:wchar_t /Zc:forScope /Fo"Debug\\" /Fd"Debug\vc120.pdb" /Gd /TC /analyze- /errorReport:prompt "源2.c"
         源2.c
       Link:
         E:\Cpp\VC\bin\link.exe /ERRORREPORT:PROMPT /OUT:"C:\Users\hp\Documents\GitHub\2018.10.7\第三次\Project7\Debug\Project7.exe" /INCREMENTAL /NOLOGO kernel32.lib user32.lib gdi32.lib winspool.lib comdlg32.lib advapi32.lib shell32.lib ole32.lib oleaut32.lib uuid.lib odbc32.lib odbccp32.lib /MANIFEST /MANIFESTUAC:"level='asInvoker' uiAccess='false'" /manifest:embed /DEBUG /PDB:"C:\Users\hp\Documents\GitHub\2018.10.7\第三次\Project7\Debug\Project7.pdb" /TLBID:1 /DYNAMICBASE /NXCOMPAT /IMPLIB:"C:\Users\hp\Documents\GitHub\2018.10.7\第三次\Project7\Debug\Project7.lib" /MACHINE:X86 "Debug\源.obj"
         "Debug\源1.obj"
         "Debug\源2.obj"
     1>源1.obj : error LNK2005: _main 已经在 源.obj 中定义
     1>源2.obj : error LNK2005: _main 已经在 源.obj 中定义
     1>C:\Users\hp\Documents\GitHub\2018.10.7\第三次\Project7\Debug\Project7.exe : fatal error LNK1169: 找到一个或多个多重定义的符号
     1>已完成生成项目“C:\Users\hp\Documents\GitHub\2018.10.7\第三次\Project7\Project7\Project7.vcxproj”(Build 个目标)的操作 - 失败。

生成失败。

已用时间 00:00:00.75
